apps/vessel-history/features/terminology/__snapshots__/terminology-fishing-events.spec.tsx.snap

```diff
@@ -27,7 +27,14 @@ exports[`TerminologyFishingEvents should render successfully 1`] = `
       >
         https://globalfishingwatch.org/datasets-and-code/
       </a>
-      .
+      and FAQs:
+      <a
+        href="events.fishingEventsFaqsLink"
+        rel="noopener noreferrer"
+        target="_blank"
+      >
+        https://drive.google.com/file/d/1N4YRJ_yxAObEIWbaYM8l-YDPHS0bKe2N/view?usp=sharing/
```

Comment:
Is this the new link for the updated FAQs? Leah said she had to update the doc in drive and that's why she blocked ths story.

Reply:
no @rdgfuentes , but now she can modify it in the translation

Reply:
Perfect